cypress/integration/change-metadata.spec.ts

```diff
@@ -1,11 +1,18 @@
 describe("Changing metadata in the language info page", function () {
+  beforeEach(() => {
+    cy.clearLocalData();
+    cy.generateProject();
+  });
+
   it.skip("should find a button to press to add source by uploading file", function () {
-    const languageName = "Makah";
+    const languageName = "’Are’are";
```

Comment:
Should this test case still be skipped?

Reply:
Good catch!

Reply:
The test seems to fail here, as the changes made here don't persist when the page reloads. I can't explain why this would be the case

Reply:
This test should work now